### PR TITLE
Run Active Storage tests in CI

### PR DESCRIPTION
**Pull request by georgeclaghorn**

### CODE REVIEW

**Comment by georgeclaghorn** [comment]
I think this is ready to be merged. We’ll also want to run the S3, GCS, and Azure service tests in master, but I’ll set that up separately because it’ll take a little more work.

r? @rafelfranca

File: activestorage/test/test_helper.rb [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
+
+begin
+  require "byebug"
+rescue LoadError
+end
 
 require "active_job"
 ActiveJob::Base.queue_adapter = :test
-ActiveJob::Base.logger = nil
+ActiveJob::Base.logger = ActiveSupport::Logger.new(nil)
```
Comment (rafaelfranca) [inline activestorage/test/test_helper.rb]: Does not setting the `logger` to `nil` works?

File: activestorage/test/test_helper.rb [hunk 2]
```diff
@@ -4,11 +4,15 @@
 require "active_support"
 require "active_support/test_case"
 require "active_support/testing/autorun"
-require "byebug"
+
+begin
+  require "byebug"
+rescue LoadError
```
Comment (rafaelfranca) [inline activestorage/test/test_helper.rb]: Why this? Maybe we should remove this require all together.

Comment (georgeclaghorn) [inline activestorage/test/test_helper.rb]: Byebug doesn't get installed in CI. This matches what Action Cable does:

https://github.com/rails/rails/blob/8f44d1661981a57838543b8a9890c855d7b02b89/actioncable/test/test_helper.rb#L10-L13

File: activestorage/test/test_helper.rb (hunk 1 above)
Comment (georgeclaghorn) [inline activestorage/test/test_helper.rb]: Nope. Active Job assumes it has a logger.

Comment (rafaelfranca) [inline activestorage/test/test_helper.rb]: Oh, that is bad. Something to change latter.

**Comment by rafaelfranca** [comment]
Nice work George!

**Pull request merged**